brave/src/test/java/com/linecorp/armeria/server/brave/BraveServiceIntegrationTest.java

```diff
@@ -123,6 +124,24 @@ public void supportsPortableCustomization() throws Exception {
                 .containsEntry("response_customizer.is_span", "false");
     }
 
+    @Override
+    @Test
+    public void options() throws Exception {
+        assertThat(HttpClient.of(url("/")).options("/").aggregate().join().status()).isEqualTo(OK);
+        final Span span = takeSpan();
+        assertThat(span.tags()).containsEntry("http.method", "OPTIONS")
+                               .containsEntry("http.path", "/");
+        assertThat(span.name()).isEqualTo("options exact:/");
+    }
+
+    @Override
```

Comment:
comment why overriding? we don't really want to override without calling super unless there's something odd. It is fine to add new methods, but overriding causes questions.